i18n/en/docusaurus-plugin-content-docs/version-5.0/12-observability/01metrics.md

```diff
@@ -0,0 +1,102 @@
+| gauge | rocketmq_consumer_inflight_messages | count | The number of inflight messages. | cluster,node_type,node_id,topic, consumer_group |
+| gauge | rocketmq_consumer_queueing_latency | millisecond | Ready messages queueing delay time. | cluster,node_type,node_id,topic, consumer_group |
+| gauge | rocketmq_consumer_lag_latency | millisecond | The delayed time before messages are consumed. | cluster,node_type,node_id,topic, consumer_group |
+| counter | rocketmq_send_to_dlq_messages_total | count | The number of messages that are sent to the dead-letter queue. | cluster,node_type,node_id,topic, consumer_group |
+| histogram | rocketmq_rpc_latency | millisecond | The rpc call latency | cluster,node_typ,node_id,protocol_type,request_code,response_code |
+| gauge | rocketmq_storage_size | byte | The size of the storage space that is used by the node. | cluster,node_type,node_id |
+| counter | rocketmq_storage_read_bytes_total | byte | The amount of data read by the storage layer. | cluster,node_type,node_id,topic |
+| gauge | rocketmq_storage_read_bytes_max | byte | Peak data read per second of the storage layer. | cluster,node_type,node_id,topic |
+| counter | rocketmq_storage_write_bytes_total | byte | The amount of data write to the storage layer. | cluster,node_type,node_id,topic |
+| gauge | rocketmq_storage_write_bytes_max | byte | Peak data write per second to the storage layer. | cluster,node_type,node_id,topic |
+| Histogram | rocketmq_storage_write_latency | millisecond | The latency of messages sizes. This metric is counted only when messages are sent. The following shows the distribution ranges:<br />le_1_ns: ≤ 1 ns<br /> le_10_ns: ≤ 10 ns<br />le_100_ns: ≤ 100 ns<br />le_1000_ns: ≤ 1 ms<br />le_overflow: > 1 ms  | cluster,node_type,node_id,topic,message_type |
+| gauge | rocketmq_storage_message_reserve_time | millisecond | Message retention time. | cluster,node_type,node_id |
+| gauge | rocketmq_storage_dispatch_behind_bytes | byte | Undispatched message size. | cluster,node_type,node_id |
+| gauge | rocketmq_storage_flush_behind_bytes | byte | Unflushed messsage size. | cluster,node_type,node_id |
+| gauge | rocketmq_thread_pool_wartermark | count | The number of tasks queued in the thread pool. | cluster,node_type,node_id,name |
+
+
+
+### Producer metrics
+The following table describes the labels of the metrics that are related to the producers in Message Queue for Apache RocketMQ.
+
+- cluster: RocketMQ cluster name.
+- node_type: the type of service node, whitch includes the following:proxy,broker,nameserver.
+- node_id:the ID of the service node.
+- topic: the topic of Message Queue for Apache RocketMQ.
+- message_type: the type of a message, which includes the following:<br />Normal:normal messages;<br />FIFO:ordered messages;<br />Transaction:Transactional messages;<br />Delay:scheduled or delayed messages.
```

Comment:
I think the value of label message_type is lowercase.

https://github.com/apache/rocketmq/blob/c9975961f81292f62c2c47f9e58f64472653ea7e/common/src/main/java/org/apache/rocketmq/common/attribute/TopicMessageType.java#L60-L62